### PR TITLE
Provide ready-to-run init scripts for packages

### PR DESCRIPTION
**Pull request by daniviga**
The plan is:
## RHEL/Fedora
- [x] use systemd directly

https://ci.openquake.org/job/redhat_oq-engine/28/

### Trusty and Precise
- [x] use supervisord

https://ci.openquake.org/job/zdevel_oq-engine/1815/

### Xenial
(and unofficial support for Debian Jessie)
use supervisord for now then migrate to systemd
https://ci-dev.openquake.org/job/zdevel_oq-engine/1764/

### CODE REVIEW

**Comment by micheles** [comment]
LGTM

File: rpm/python-oq-engine.spec.inc [hunk 1]
```diff
@@ -22,7 +22,7 @@ Patch1: openquake.cfg.patch
 
 Requires(pre): shadow-utils
 
-%define common_deps python numpy scipy python-shapely python-psutil python-mock python-futures h5py python-amqp python-celery python-django python-docutils rabbitmq-server sudo
+%define common_deps sudo systemd python numpy scipy python-shapely python-psutil python-mock python-futures h5py python-amqp python-celery python-django python-docutils python-gunicorn rabbitmq-server
```
Comment (nastasi-oq) [inline rpm/python-oq-engine.spec.inc]: Why sudo is a requirement now ?

**Comment by nastasi-oq** [comment]
Except the question about 'sudo' (just for curiosity) LGTM.

File: rpm/python-oq-engine.spec.inc (hunk 1 above)
Comment (daniviga) [inline rpm/python-oq-engine.spec.inc]: It was already a requirement: lots of our scripts use sudo to change the user to openquake. I just moved the requirement from the end of the line to the start to have a more logic order in the dependencies

**Comment by daniviga** [comment]
I'm investigating an issue with Supervisord on Xenial (see https://ci-dev.openquake.org/job/zdevel_oq-engine/1763/), I'll wait Monday before merging this.

File: rpm/python-oq-engine.spec.inc (hunk 1 above)
Comment (nastasi-oq) [inline rpm/python-oq-engine.spec.inc]: Oh, a replacement of 'sudo' wasn't in my options list. Ok.

**Comment by nastasi-oq** [comment]
LGTM

**Comment by daniviga** [comment]
New tests: https://ci.openquake.org/job/zdevel_oq-engine/1826/ and https://ci-dev.openquake.org/job/zdevel_oq-engine/1766/

**Comment by daniviga** [comment]
New tests: https://ci.openquake.org/job/zdevel_oq-engine/1827/ and https://ci-dev.openquake.org/job/zdevel_oq-engine/1767/

**Pull request merged**